tests/Queue/RedisQueueIntegrationTest.php

```diff
@@ -65,6 +65,31 @@ public function testExpiredJobsArePopped($driver)
         $this->assertEquals(3, $this->redis[$driver]->connection()->zcard('queues:default:reserved'));
     }
 
+    /**
+     * @dataProvider redisDriverProvider
+     *
+     * @param $driver
+     *
+     * @throws \Exception
+     */
+    public function testBlockingPop($driver)
+    {
+        $this->tearDownRedis();
+        if ($pid = pcntl_fork() > 0) {
```

Comment:
I don't think the `pcntl` extension is required.  You might need to check if it's installed and skip the test if it's not.